lib/runtime.coffee

```diff
@@ -65,11 +65,16 @@ class Runtime
   # Public: stops execution of the current fork
   stop: ->
     @runner.stop()
+    @emitter.emit 'did-execute-stop'
 
   # Public: Dispatched when the execution is starting
   onDidExecuteStart: (callback) ->
     @emitter.on 'did-execute-start', callback
 
+  # Public: Dispatched when the execution is stopped
+  onDidExecuteStop: (callback) ->
+    @emitter.on 'did-execute-stop', callback
```

Comment:
Agreed. The intention here though was to align the name with the existing codebase. So if you don't mind, i'll also change the start event/function names to reflect that and add proper pre- and post events/functions which might come in handy later.